scripts/tests/assertoor_network_params.yaml

```diff
@@ -0,0 +1,23 @@
+# Full configuration reference [here](https://github.com/ethpandaops/ethereum-package?tab=readme-ov-file#configuration).
+participants:
+  - el_type: geth
+    el_image: ethereum/client-go:latest
+    cl_type: lighthouse
+    cl_image: lighthouse:local
+    cl_extra_params:
+      - --target-peers=3
+    count: 4
+network_params:
+  deneb_fork_epoch: 0
+  seconds_per_slot: 6
+  num_validator_keys_per_node: 20
+global_log_level: debug
+snooper_enabled: false
+additional_services: 
+  - assertoor
+assertoor_params:
+  run_stability_check: true
+  run_block_proposal_check: true
+  run_blob_transaction_test: true
+  tests:
+    - https://raw.githubusercontent.com/ethpandaops/assertoor-test/master/assertoor-tests/big-calldata-tx-test.yaml
```

Comment:
Looks like the `big-calldata-tx-test.yaml` is the one taking the most time (>20m) and doesn't run concurrently with the other 3 tests. Maybe we can exclude this one from the CI job and have it as a recurring test on GitHub action?